amads/melody/segment_gestalt.py

```diff
@@ -78,10 +78,9 @@ def construct_score_list(notes, intervals):
     score_list = []
```

Comment:
Thinking out loud, I guess under the new proposal we can write this function as:

```py
return [Score(Part(note.copy() for note in notes[interval[0] : interval[1]])) for interval in intervals]
```

I think it's a good sign that we can write it as a one-liner -- shows that we're writing Pythonic code.

Reply:
I think that's right -- I'm not sure I'd want to read one-liners with that much going on, but yes.